index.html

```diff
@@ -226,37 +219,31 @@ <h1>Proof Methods</h1>
 
     <tr>
       <td>
-          EcdsaSecp256k1Signature2019
+EcdsaSecp256k1Signature2019
+(<a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">spec</a>)
       </td>
       <td>
-          <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">
-            EcdsaSecp256k1Signature2019</a>
+        Community Draft
       </td>
       <td>
         None
       </td>
-      <td>
-        Reserved in https://www.w3.org/2018/credentials/v1
-      </td>
       <td>
         None
       </td>
     </tr>
 
     <tr>
       <td>
-          EcdsaSecp256r1Signature2019
+EcdsaSecp256r1Signature2019
+(<a href="https://w3c-dvcg.github.io/lds-ecdsa-secp256r1-2019/">spec</a>)
```

Comment:
```suggestion
(<a href="https://w3c-ccg.github.io/di-ecdsa-secpr1-2019/">spec</a>)
```
via https://github.com/w3c-ccg/community/issues/228